Git Formats/Git Log.sublime-syntax

```diff
@@ -13,13 +13,33 @@ contexts:
   main:
     # 1st header line
     # commit d9d9fb804f5d61c13ba2f8746af33a9f3c609075
-    - match: (?:(commit)\s+(\h{7,}))?\s*\n
+    - match: \b(commit)\s+(\h{7,})
       scope: meta.header.git.commit markup.raw.block.git.log
       captures:
         1: keyword.other.commit.git.log
         2: constant.other.hash.git.log
-      embed: commit-header
-      escape: (?=^commit\s)
+      push:
+        - match: $\n?
+          scope: meta.header.git.commit markup.raw.block.git.log
+          embed: commit-header
+          escape: (?=^commit\s)
+          pop: true
+        - match: \s*(\()(HEAD)\s*(->)
+          captures:
+            1: punctuation.section.parens.begin.git.log
+            2: support.type.git.log
+            3: punctuation.separator.mapping.git.log
+          push:
+            - meta_scope: meta.header.git.commit markup.raw.block.git.log
```

Comment:
If we move this up one level, the `scope` of the match in line 16 can be removed.

Reply:
done :)